src/main/resources/templates/fragments/layout.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout" lang="en"
+      xmlns:th="http://www.thymeleaf.org" class="govuk-template">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="content-type" content="text/html; charset=UTF-8"/>
+
+  <title layout:title-pattern="$LAYOUT_TITLE - $CONTENT_TITLE"
+         th:text="${(param.error != null || error != null) ? 'Error: ' : ''} + 'HMPPS Digital Services'">
+    HMPPS Digital Services
+  </title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <meta name="theme-color" content="#0b0c0c"/>
+
+  <!-- Google Tag Manager -->
+  <script th:if="${@environment.getProperty('application.google.tag.id')} != null">(function(w, d, s, l, i) {
```

Comment:
I wonder if anything bad will happen if we have this GA stuff

Reply:
Good catch - I'll remove in the next PR.